pyramid_oereb/contrib/print_proxy/mapfish_print.py

```diff
@@ -324,6 +327,14 @@ def __call__(self, value, system):
             del response.headers['Connection']
         return content
 
+    def _clean_other_legend(self, other_legends, legend_element):
+        duplicata_index = []
+        for other_legend in other_legends:
+            if other_legend['SymbolRef'] == legend_element['SymbolRef']:
+                duplicata_index.append(other_legends.index(other_legend))
+        for index in duplicata_index:
+            del other_legends[index]
```

Comment:
I do not think this is going to have the desired effect. After the "del", the indexes won't be the same so we can not continue in the for loop (it will delete the wrong elements or be out of bounds).

Reply:
You're right, thanks